gef.py

```diff
@@ -1231,27 +1235,27 @@ def __iter__(self):
             yield current_arena
         return
 
-    def __eq__(self, other):
+    def __eq__(self, other) -> bool:
         # You cannot have 2 arenas at the same address, so this check should be enough
         return self.__addr == int(other)
 
-    def fastbin(self, i):
+    def fastbin(self, i: int):
```

Comment:
return type?

Reply:
would be `Optional[GlibcChunk]` but it's not possible to add this here without moving the definition of `GlibcChunk` before `GlibcArena` and iirc `GlibcChunk` also has to reference the type of `GlibcChunk` (-> what I called "circular type hints")

Reply:
Can't we add the comment like you did below?

Reply:
yes, that would be possible!